### ISSUE TITLE
Create Temporary SVG Dot Drawing Helper

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#4]
## Background
While pure HTML and CSS may be enough for simple graphics like bar and column charts, we'll need a slightly better format for charts that require curves, circles, triangles, and so on.

The format in question is none other than **SVG**, which stands for _Scalable Vector Graphics_. You may have seen images with the `.svg` extension before - that means you can scale them up and down without losing quality! This makes SVGs a popular choice for logos and icons, but in our case, they can also be used for scalable charts.

### How Do They Work?
SVGs are defined using _paths_. You can think of your web browser (or whatever application you're using) as a painter, and the SVG file as a set of instructions that specify which brush strokes they should use to draw an image.

[Here](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths) is a very good article on SVG paths, and you'll definitely want to read through it before starting the issue.

## Goal
Instead of building a chart using SVG paths right away, we will tackle a much smaller challenge - creating a helper that renders an SVG graphic. For more information on helpers in Ruby on Rails, see issues #1 and #2.

Please name it `dot_svg_render` and use the information from the article above to render an SVG that looks something like this -

<img width="438" alt="image" src="https://github.com/PureChart/purechart/assets/43429820/b5016cd6-cbff-4cf0-9673-7d637b76afce">

It doesn't have to be exact - just some dots scattered around will do. Good luck!

**Comment by GeorgeBerdovskiy** [comment]
SVG code that looks like this...
```html
<svg width="100" height="100" xmlns="http://www.w3.org/2000/svg" style="border: 1px dashed lightgray;">
  <!-- P1 -->
  <circle cx="calc(0px + 3px)" cy="calc(100px - 0px - 3px)" r="3" fill="red" name="P1"/>
 
  <!-- P2 -->
  <circle cx="calc(50px + 3px)" cy="calc(100px - 75px - 3px)" r="3" fill="red"/>
  
  <!-- P3 -->
  <circle cx="calc(60px + 3px)" cy="calc(100px - 80px - 3px)" r="3" fill="red"/>
</svg>
 ```

... will look like this when rendered!
<img width="271" alt="image" src="https://github.com/PureChart/purechart/assets/43429820/bf5d4ddb-6690-4ca0-b169-9c0f22e80f99">

Check out [W3Schools](https://www.w3schools.com/graphics/tryit.asp?filename=trysvg_myfirst) for interactive SVG tutorials you can play around with in your web browser.

**Comment by viveknarayana** [comment]
I'll work on this

**Comment by GeorgeBerdovskiy** [comment]
<img width="794" alt="image" src="https://github.com/PureChart/purechart/assets/43429820/bd09397b-c252-4ffb-b258-1c9d7a6fa290">
Closed by #9.
